model/schema/allele.yaml

```diff
@@ -324,12 +482,25 @@ slots:
     description: >-
```

Comment:
'primary_image' slot range is not specified; seems that it should be a boolean

Reply:
Boolean seems sensible, but it’s just carried over from the current repo - not a new slot - so I’m not sure.  There’s definitely tidying up to do on the allele model but the aim of this PR was simply to transition to a model with the new paradigm of separate ingest classes.

Reply:
Are you happy enough with the new parts of the model that we can merge @chris-grove ?